server/src/main/java/engine/ActiveGame.java

```diff
@@ -40,7 +41,14 @@ public GameRunnerState run() throws IOException {
       ShotResult result = round.fireShot(fleetUnderFire, shot);
       logShotInfo(shot, result);
 
-      if (result == ShotResult.HIT) {
+      if (result == ShotResult.SUNK) {
```

Comment:
It does not look good.

Reply:
We still don't know how to refactor and test this class